tad/api/routes/projects.py

```diff
@@ -0,0 +1,56 @@
+import logging
+from typing import Annotated
+
+from fastapi import APIRouter, Depends, Request
+from fastapi.responses import HTMLResponse
+
+from tad.api.deps import templates
+from tad.core.config import get_settings
+from tad.core.exceptions import UnsafeFileError
+from tad.schema.instrument import InstrumentBase
+from tad.schema.project import ProjectNew
+from tad.schema.system_card import SystemCard
+from tad.services.instruments import InstrumentsService
+from tad.services.projects import ProjectsService
+from tad.services.storage import WriterFactory
+from tad.utils.path import is_safe_path
+
+router = APIRouter()
+logger = logging.getLogger(__name__)
+
+
+@router.get("/new")
+async def get_new(
+    request: Request,
+    instrument_service: Annotated[
+        InstrumentsService, Depends(InstrumentsService)
+    ],  # todo: create async_instruments_service
+) -> HTMLResponse:
+    instruments = instrument_service.fetch_instruments()
+
+    return templates.TemplateResponse(request, "projects/new.html.j2", {"instruments": instruments})
+
+
+@router.post("/new", response_class=HTMLResponse)
+async def post_new(
+    request: Request,
+    project_new: ProjectNew,
+    projects_service: Annotated[ProjectsService, Depends(ProjectsService)],
+) -> HTMLResponse:
+    system_card_file = get_settings().CARD_DIR / f"{project_new.name}_system.yaml"
```

Comment:
Name's aren't unique, are they? Wouldn't it be better to use the id?

Reply:
agreed. will change it

Reply:
@anneschuth. What do you think about an id and a name? think this would help if we start writing it to remove locations for usability

Reply:
The issue is that we did not have the ID of the project because the project is not inserted into the database. I will need to change the model that we can insert a project without a system card. will apply that now.